android-base/src/main/java/io/github/droidkaigi/confsched2020/di/AppComponent.kt

```diff
@@ -1,41 +1,43 @@
-package io.github.droidkaigi.confsched2020.di
-
-import android.app.Application
-import dagger.BindsInstance
-import dagger.Component
-import dagger.android.AndroidInjector
-import dagger.android.support.AndroidSupportInjectionModule
-import io.github.droidkaigi.confsched2020.App
-import io.github.droidkaigi.confsched2020.MainActivityModule
-import io.github.droidkaigi.confsched2020.model.repository.ContributorRepository
-import io.github.droidkaigi.confsched2020.model.repository.SessionRepository
-import io.github.droidkaigi.confsched2020.model.repository.StaffRepository
-import javax.inject.Singleton
-
-@Singleton
-@Component(
-    modules = [
-        AppModule::class,
-        AndroidSupportInjectionModule::class,
-        MainActivityModule.MainActivityBuilder::class,
-        DbComponentModule::class,
-        RepositoryComponentModule::class,
-        FirestoreComponentModule::class,
-        ApiComponentModule::class,
-        DeviceComponentModule::class
-    ]
-)
-interface AppComponent : AndroidInjector<App>, AppComponentInterface {
-    @Component.Factory
-    interface Factory {
-        fun create(@BindsInstance application: Application): AppComponent
-    }
-
-    override fun inject(app: App)
-
-    override fun sessionRepository(): SessionRepository
-    fun staffRepository(): StaffRepository
-    fun contributorRepository(): ContributorRepository
-}
-
-fun Application.createAppComponent() = DaggerAppComponent.factory().create(this)
+package io.github.droidkaigi.confsched2020.di

```

Comment:
file /Users/takahirom/Desktop/AppComponent.kt.txt
/Users/takahirom/Desktop/AppComponent.kt.txt: ASCII text, with CRLF line terminators

Probably it is not fixed yet 😭

Reply:
Sorry for making many CRLFs... I fixed.